src/app/components/shared/footer/footer.component.scss

```diff
@@ -1,67 +1,73 @@
- .footer {
-  max-width: 100%;
+* {
+	margin: 0;
+	padding: 0;
+  margin-top: 6px;
+}
+
+.footer {
   background-color: #787878;
+  width: 100%;
+}
 
- }
-  h3 {
-    margin-bottom: 24px;
-  }
-  
-  .container {
-    box-sizing: border-box;
-    margin: auto;
-    padding: 0;
-    max-width: 1170px;
-    min-width: 300px;
-    font-weight: normal;
-    display: flex;
-    justify-content: center;
-  }
-  
-  .footer--blocks_contacts {
-    margin-top: 22px;
-  }
-  
-  .footer-icons {
-    margin-top: 20px;
-  }
-  
-  .pi-envelope {
-    margin-right: 11px;
+
+
+h3 {
+  margin-bottom: 24px;
+}
+
+.container {
+  box-sizing: border-box;
+  margin: auto;
+  padding: 0;
+  max-width: 1170px;
+  min-width: 320px;
+}
+
+.footer--blocks_contacts {
+  margin-top: 22px;
+}
+
+.footer-icons {
+  margin-top: 2px;
+}
+
+.pi-envelope {
+  margin-right: 11px;
+}
+
+.footer-icons a {
+  text-decoration: none;
+  cursor: pointer;
+}
+.img-footer {
+  width: 100%;
+  height: auto;
+}
+
+.footer-text {
+  margin-top: 26px;
+}
+
+@media screen and (max-width: 575px) {
+  .footer-blocks {
+    justify-content: space-around;
   }
-  
-  .footer-icons a {
-    text-decoration: none;
-    cursor: pointer;
+
+  .footer-contacts {
+    width: 100% !important;
+    margin: 10px 10px 30px 10px !important;
+    order: 0 !important;
   }
-  .img-footer {
-    width: 135%;
+
+  .footer-img {
+    width: 100% !important;
+    margin: 10px !important;
+    order: 2 !important;
   }
-  
-  .footer-text {
-    margin-top: 20px;
+
+  .footer-dev {
+    width: 100% !important;
```

Comment:
to remove important use one approach
because now you are using to different:
 1. primeflex.css which use !important 
      .w-4 {
         width: 33.3333% !important;   
       }
2. @media screen and (max-width: 575px) { 
      there you are trying to override primeflex

Reply:
Fixed